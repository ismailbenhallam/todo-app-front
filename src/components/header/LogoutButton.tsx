import styled from "styled-components";

const Button = styled.button`
  width: 140px;
  height: 36px;
  border-radius: 4px;
  border-color: #eb514e;
  font-family: "Lato";
  font-size: 14px;
  color: #eb514e;
  background-color: transparent;
  cursor: pointer;
`;

export default function LogoutButton() {
  return <Button>DÃ©connexion</Button>;
}
