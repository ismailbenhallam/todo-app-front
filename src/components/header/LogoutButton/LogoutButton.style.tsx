import { Button } from "@mui/material";
import styled from "styled-components";

export const ButtonContainer = styled(Button)`
  width: 200px;
  height: 36px;
  border-radius: 4px;
  border-color: #eb514e;
  font-family: "Lato";
  font-size: 14px;
  color: #eb514e;
  background-color: transparent;
  cursor: pointer;
`;
