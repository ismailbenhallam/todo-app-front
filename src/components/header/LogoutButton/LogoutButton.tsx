import { FormattedMessage } from "react-intl";
import { Button } from "./LogoutButton.style";

export default function LogoutButton() {
  return (
    <Button>
      <FormattedMessage id="buttons.logout" />
    </Button>
  );
}
