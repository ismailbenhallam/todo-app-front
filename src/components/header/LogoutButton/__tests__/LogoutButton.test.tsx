import renderer from "react-test-renderer";
import LogoutButton from "..";

test("LogouButton should match snapshot", () => {
  const tree = renderer.create(<LogoutButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
