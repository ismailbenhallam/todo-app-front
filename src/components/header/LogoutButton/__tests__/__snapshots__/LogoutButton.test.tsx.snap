// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogouButton should match snapshot 1`] = `
<button
  className="sc-bdfBQB jPjxVN"
>
  DÃ©connexion
</button>
`;
